batch/create/create_gpu_job.js

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+async function main() {
+  // [START batch_create_gpu_job]
```

Comment:
We need to have one more sample with `batch_create_gpu_job_n1` region tag, to show how to use T4 and other GPUs that run on N1 platform.

Python version: https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/batch/create/create_with_gpu_no_mounting.py

Please add the N1 sample to this PR as well, as they're a "pair" :)